Weaning.Rmd

```diff
@@ -0,0 +1,39 @@
+---
+title: "Weaning Paper Summaries"
+author: "Farida Samad-zada"
+date: '27 ноября 2017 г '
+output: html_document
+---
+In my section of introduction, I focus on post-birth resource allocations by the mother.
+
+##Classic Paper on Parent-Offspring Conflict by R.L. Trivers 
```

Comment:
Don't forget the space between the heading indicator (`#`) and the heading text. So in this case it should be:

    ## Classic Paper on Parent....